host_classifier.sh

```diff
@@ -25,29 +30,23 @@
  lin_os_count=0
  other_os_count=0
 
-#####Primary Script#####
 
-#1stWhile
-#The "1stWhile" loop shifts to each file input addressing the multiple files requirement.
-while [ "$1" != "" ]
- do
  #prints the current filename excluding the path.  I'm sure there's a better way.
   echo =========================
   filename=$(echo $1 | cut --delimiter "/" --fields 8)
```

Comment:
i'm not sure what you're doing here, but it is definitely causing a problem with the output... in my case it isn't printing the file name

Reply:
you should be using $@ and looping over it, like so:

for filename in "$@"; do
end